official/utils/logging/hooks_helper_test.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Tests for hooks_helper."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+from official.utils.logging import hooks_helper
+
+tf.logging.set_verbosity(tf.logging.ERROR)
+
+
+class BaseTest(tf.test.TestCase):
+
+  def test_raise_in_non_list_names(self):
+    with self.assertRaises(ValueError):
+      hooks_helper.get_train_hooks(
+          'LoggingTensorHook, ProfilerHook', batch_size=256)
+
+  def test_raise_in_invalid_names(self):
+    invalid_names = ['StepCounterHook', 'StopAtStepHook']
+    with self.assertRaises(ValueError):
+      hooks_helper.get_train_hooks(invalid_names, batch_size=256)
+
+  def get_train_hook_valid_name(self, test_name, hook_name, **kwargs):
```

Comment:
For test_name, I think it should be more readable to as "test_hook_name", and also be the type of string, instead of a list. You can wrap it as a list and pass it to hooks_helper.get_train_hooks